app/models/student.rb

```diff
@@ -1,5 +1,5 @@
 class Student < ApplicationRecord
   belongs_to :user
   has_many :students_groups
-  has_many :groups, through: :students_groups
+  has_many :study_groups, through: :students_groups
```

Comment:
What's a study_group? O.o

Reply:
yes I made a big mistake in the morning and generated a model with a wrong name, I didn't finish cleaning up after it, thanks for finding this